lib/mayaUsd/nodes/layerManager.cpp

```diff
@@ -448,7 +458,10 @@ bool LayerDatabase::saveUsd(bool isExport)
     auto opt = MayaUsd::utils::serializeUsdEditsLocationOption();
 
     if (MayaUsd::utils::kIgnoreUSDEdits != opt) {
-        if (_batchSaveDelegate && _proxiesToSave.length() > 0) {
+        if (isCrashing()) {
+            result = kPartiallyCompleted;
+            opt = MayaUsd::utils::kSaveToUSDFiles;
```

Comment:
Your pull request description says you'll save inside the Maya file when crashing, but your code says save to USD files.  Which one should it be?

Reply:
Is Maya, thanks for catching this.